### PR TITLE
Add activity package

### PR DESCRIPTION
**Pull request by oscarsu97**
Contains name and location class

### CODE REVIEW

File: src/main/java/seedu/address/commons/core/Config.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ public int hashCode() {
     public String toString() {
         StringBuilder sb = new StringBuilder();
         sb.append("Current log level : " + logLevel);
-        sb.append("\nPreference file Location : " + userPrefsFilePath);
+        sb.append("\nPreference file Address : " + userPrefsFilePath);
```
Comment (1nefootstep) [inline src/main/java/seedu/address/commons/core/Config.java]: How about file path?

File: src/main/java/seedu/address/model/activity/Activity.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+     */
+    public Set<Tag> getTags() {
+        return Collections.unmodifiableSet(tags);
+    }
+
+    /**
+     * Returns true if both persons of the same name have at least one other identity field that is the same.
+     * This defines a weaker notion of equality between two persons.
+     */
+    public boolean isSameActivity(Activity otherActivity) {
```
Comment (1nefootstep) [inline src/main/java/seedu/address/model/activity/Activity.java]: Why is this necessary compared to just having equals?

File: src/main/java/seedu/address/model/activity/Activity.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        if (otherActivity == this) {
+            return true;
+        }
+        return otherActivity != null
+                && otherActivity.getName().equals(getName())
+                && otherActivity.getLocation().equals(getLocation());
+    }
```
Comment (KxxMxxx) [inline src/main/java/seedu/address/model/activity/Activity.java]: Is there a need for this method? It seems to me that you're checking whether the activity has the same name and location but different tags. Is there a case where this method would come to use?
